tests/behat/features/web/translation/translate_project.feature

```diff
@@ -7,29 +7,112 @@ Feature: Project title, description and credits should be translatable via a but
+    And the "#description-translation" element should contain "translated mydescription"
+    And the element "#credits-translation" should be visible
+    And the "#credits-translation" element should contain "translated mycredit"
+    When I click "#remove-program-translation-button"
+    Then the element "#program-translation-button" should be visible
+    And the element "#name" should be visible
+    And the element "#description" should be visible
+    And the element "#credits" should be visible
+
+  Scenario: Translate button should translate title, description, and credit only once
+    Given I am on "/app/project/1"
+    And I wait for the page to be loaded
+    Then the element "#program-translation-button" should exist
+    When I click "#program-translation-button"
+    And I wait for AJAX to finish
+    Then the element "#remove-program-translation-button" should be visible
+    And the element "#name-translation" should be visible
+    When I click "#remove-program-translation-button"
+    Then the element "#program-translation-button" should be visible
+    And the element "#name" should be visible
+    When I click "#program-translation-button"
+    Then the element "#remove-program-translation-button" should be visible
+    And the element "#name-translation" should be visible
+
+  Scenario: Translation should show translated by line when credits available
+    Given I am on "/app/project/4"
+    And I wait for the page to be loaded
+    Then the element "#program-translation-button" should exist
+    When I click "#program-translation-button"
+    And I wait for AJAX to finish
+    Then the element "#credits-translation-wrapper" should be visible
+    And the "#program-translation-before-languages" element should contain "Translated by iTranslate from"
+    And the "#program-translation-first-language" element should contain "English"
+    And the "#program-translation-between-languages" element should contain "to"
+    And the "#program-translation-second-language" element should contain "English"
+    And the "#program-translation-after-languages" element should contain ""
+  
+  Scenario: Translation should show translated by line when credits not available
+    Given I am on "/app/project/1"
+    And I wait for the page to be loaded
+    Then the element "#program-translation-button" should exist
+    When I click "#program-translation-button"
+    And I wait for AJAX to finish
+    Then the element "#credits-translation-wrapper" should be visible
+    And the "#program-translation-before-languages" element should contain "Translated by iTranslate from"
+    And the "#program-translation-first-language" element should contain "English"
+    And the "#program-translation-between-languages" element should contain "to"
+    And the "#program-translation-second-language" element should contain "English"
+    And the "#program-translation-after-languages" element should contain ""
```

Comment:
if you remove the duplicate byline in the twig template, then these 2 tests can be merged. But i'm not sure if there is a good reason that you duplicated that byline.

Reply:
I removed this test and added another one to make sure the description and credits were not replaced when they do not exist